src/app/types/self-subject-access-review.ts

```diff
@@ -0,0 +1,20 @@
+export class SelfSubjectAccessReview {
+  readonly kind = 'SelfSubjectAccessReview';
+  readonly apiVersion = 'authorization.k8s.io/v1';
+  readonly spec = {
+    resourceAttributes: {
+      namespace: 'default',
+      verb: '',
+      resource: '',
+    },
+  };
+  readonly status = {
+    allowed: false,
+    reason: '',
+  };
+
+  constructor(verb: string, resource: string) {
+    this.spec.resourceAttributes.verb = verb;
+    this.spec.resourceAttributes.resource = resource;
+  }
```

Comment:
Maybe also pass the group here?

Reply:
Is the group fix, or does it change?

Reply:
It can change, e.g. `organization.appuio.io`